### PR TITLE
Add keymitt ble documentation

### PR DESCRIPTION
**Pull request by spycle**
## Proposed change
Documentation for new MicroBot Push integration (keymitt_ble)

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/76575
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3585
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/keymitt_ble.markdown [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: MicroBot Push
+description: Instructions on how to set up the MicroBot Push.
+ha_category:
+  - Switch
+ha_release: ???
```
Comment (frenck) [inline source/_integrations/keymitt_ble.markdown]:
```suggestion
ha_release: 2022.8
```

File: source/_integrations/keymitt_ble.markdown [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  - switch
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+This integration allows you to locally control a MicroBot Push (previously manufactured by Naran but now under the Keymitt brand).
+
+### Prerequisites
+
+In order to use this integration, it is required to have working Bluetooth set up on the device running Home Assistant. A Naran/Keymitt hub is not required.
```
Comment (frenck) [inline source/_integrations/keymitt_ble.markdown]:
```suggestion
In order to use this integration, it is required to have working [Bluetooth](/integrations/blueooth) set up on the device running Home Assistant. A Naran/Keymitt hub is not required.
```

File: source/_integrations/keymitt_ble.markdown [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+The device will need to be in pairing mode before adding to Home Assistant. To reset the MicroBot Push, turn it off then back on and immediately hold the push button while the LED is red. After approximately 5 seconds the LED will flash rapidly at which point release the button. The LED will blink blue if pairing mode has been successfully activated.
```
Comment (frenck) [inline source/_integrations/keymitt_ble.markdown]:
```suggestion
The device will need to be in pairing mode before adding to Home Assistant. To reset the MicroBot Push, turn it off, then back on, and immediately hold the push button while the LED is red. After approximately 5 seconds, the LED will flash rapidly, at which point release the button. The LED will blink blue if the pairing mode has been successfully activated.
```

File: source/_integrations/keymitt_ble.markdown [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+If you have multiple devices, you will need to know the BTLE MAC address of your device to tell them apart.
```
Comment (frenck) [inline source/_integrations/keymitt_ble.markdown]:
```suggestion
If you have multiple devices, you will need to know the BTLE MAC address of your device to tell them apart.
```

File: source/_integrations/keymitt_ble.markdown [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+### Prerequisites
+
+In order to use this integration, it is required to have working Bluetooth set up on the device running Home Assistant. A Naran/Keymitt hub is not required.
+
+The device will need to be in pairing mode before adding to Home Assistant. To reset the MicroBot Push, turn it off then back on and immediately hold the push button while the LED is red. After approximately 5 seconds the LED will flash rapidly at which point release the button. The LED will blink blue if pairing mode has been successfully activated.
+
+If you have multiple devices, you will need to know the BTLE MAC address of your device to tell them apart. 
+
+Please note, devices cannot remain paired to the MicroBot app for this integration to function. They will be paired to Home Assistant exclusively.
```
Comment (frenck) [inline source/_integrations/keymitt_ble.markdown]:
```suggestion
Please note, that devices cannot remain paired to the MicroBot app for this integration to function. They will be paired to Home Assistant exclusively.
```

File: source/_integrations/keymitt_ble.markdown [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+### Supported Devices
+
+This Integration is for the MicroBot Push only. The Keymitt lock is not supported.
+
+### Service `keymitt_ble.calibrate`
+
+The Calibration service will locally set the MicroBot Push depth, duration, and mode.
+
+Please note: The push arm will extend or retract (dependant on the mode set) after the service call is invoked. The mode and depth will be demomstrated but not the duration. The setting is however stored and can be confirmed by manually operating the device.
```
Comment (frenck) [inline source/_integrations/keymitt_ble.markdown]:
```suggestion
Please note: The push arm will extend or retract (dependant on the mode set) after the service call is invoked. The mode and depth will be demonstrated, but not the duration. The setting is, however, stored and can be confirmed by manually operating the device.
```

File: source/_integrations/keymitt_ble.markdown [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+| Service Data Attribute | Required | Description                                                                                   |
+| ---------------------- | -------- | --------------------------------------------------------------------------------------------- |
+| `entity_id`            | yes      | Entity ID of the MicroBot Push to calibrate.                                                  |
```
Comment (frenck) [inline source/_integrations/keymitt_ble.markdown]: Isn't this an entity service? (Thus `entity_id` isn't required, one could target areas, devices or everything alternatively).

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/keymitt_ble.markdown [hunk 8]
```diff
@@ -0,0 +1,61 @@
+---
+title: MicroBot Push
+description: Instructions on how to set up the MicroBot Push.
+ha_category:
+  - Switch
+ha_release: 2022.8
```
Comment (bdraco) [inline source/_integrations/keymitt_ble.markdown]:
```suggestion
ha_release: "2022.10"
```

**Review by frenck — APPROVED** [review]

**Pull request merged**